package j;

/* renamed from: j.E, reason: case insensitive filesystem */
/* loaded from: classes.dex */
public abstract class AbstractC0739E {

    /* renamed from: a, reason: collision with root package name */
    public static final long[] f7971a = {-9187201950435737345L, -1};

    static {
        new C0769y(0);
    }

    public static final int a(int i2) {
        if (i2 == 7) {
            return 6;
        }
        return i2 - (i2 / 8);
    }

    public static final int b(int i2) {
        if (i2 == 0) {
            return 6;
        }
        return (i2 * 2) + 1;
    }

    public static final int c(int i2) {
        if (i2 > 0) {
            return (-1) >>> Integer.numberOfLeadingZeros(i2);
        }
        return 0;
    }

    public static final int d(int i2) {
        if (i2 == 7) {
            return 8;
        }
        return ((i2 - 1) / 7) + i2;
    }
}
